openff/bespokefit/schema/fitting.py

```diff
@@ -138,3 +140,26 @@ class BespokeOptimizationSchema(BaseOptimizationSchema):
         "By default bespoke torsion parameters (if requested) will be constructed for "
         "all non-terminal 'rotatable bonds'",
     )
+
+    smirk_settings: SMIRKSettings = Field(
+        SMIRKSettings(),
+        description="The settings that should be used when generating SMIRKS patterns for this optimization stage.",
+    )
```

Comment:
Do we maybe need to split this class into a bespoke and general version like we do with the fitting schemas? This seems like a field that should exist on a bespoke schema but probably not a general one!

Reply:
Currently, this is only on the `BespokeOptimizationSchema` and I don't think we have any options we want to expose like this to the general schema yet? I think for now it might be best to only keep this on the bespoke model and we can update the general one later if we need to.